src/pages/index.tsx

```diff
@@ -58,15 +73,35 @@ const HomePage: NextPageWithLayout = () => {
 
   if (signedIn || signedInOptimistic) {
     return (
-      <ComponentWrapperPage
-        src={components.tosCheck}
-        componentProps={{
-          logOut: authStore.logOut,
-          targetProps: componentProps,
-          targetComponent: components.default,
-          tosName: components.tosContent,
-        }}
-      />
+      <>
+        <VmComponent
+          src={components.nearOrg.notifications.alert}
+          props={{
+            open: showNotificationModalState,
+            handleTurnOn: () =>
+              handleTurnOn(accountId, () => {
+                setShowNotificationModalState(false);
+              }),
+            handleOnCancel: () => {
+              handleOnCancel();
+              setShowNotificationModalState(false);
+            },
+            isNotificationSupported,
+            isPermisionGranted,
+            isPushManagerSupported,
+            setNotificationsSessionStorage,
```

Comment:
To close this modal by pressing `esc` add this line:
```suggestion
            setNotificationsSessionStorage,
            onOpenChange: () => setShowNotificationModalState(false),
```